dvc/repo/get.py

```diff
@@ -56,6 +72,8 @@ def get(url, path, out=None, rev=None):
             with o.repo.state:
                 o.checkout()
 
+    except NoOutputInExternalRepoError:
```

Comment:
Btw, could you add a test for non-cached outputs, please? E.g.
```
# in external repo:
dvc run ... -O non-cached ...
...
# in repo
dvc get /path/to/erepo non-cached
```

Reply:
The option `-O` (capital o) doesn't exist in `dvc get`. Do you mean `-o`?

What is a non-cached output in this context?

Reply:
I'm talking about `-O` for `dvc run`. It means that the output is not cached, but registered as an output to a dvc-file. It is often used for some small git-tracked files, like metrics.

Reply:
@danihodovic Here is a runnable example:
```
#!/bin/bash

set -x
set -e

rm -rf mytest
mkdir mytest
cd mytest

mkdir erepo
pushd erepo
git init
dvc init
dvc run -O foo 'echo foo > foo'  # note that foo is not cached by dvc, hence the "-O"
git add .
git commit -m "init"
popd

mkdir repo
pushd repo
git init
dvc init
dvc get ../erepo foo
popd
```